merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -808,6 +832,20 @@ declare global {
      * If any spans overlap or touch, they will be coalesced into a single window.
      */
     public windows(): Windows;
+
+
+    /**
+     * Applies an expression for each instance of an activity type.
+     *
+     * @param activityType activity type to check
+     * @param expression function of an activity instance that returns a Spans
+     * @constructor
```

Comment:
please update this (and the implementation's comment) to describe that it produces Spans with associated activity IDs.